scripts/prism_4km_monthly_modern.py

```diff
@@ -34,14 +37,14 @@ def download(self, engine=None, debug=False):
         Script.download(self, engine, debug)
 
         clim_vars = ['ppt', 'tmax', 'tmean', 'tmin']
-        years = range(1981, 2015)
+        years = list(range(1981, 2015))
         months = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']
```

Comment:
I don't see why these need to be here and not just on the `for` loop. For `months`, use `range(1, 13)` and format using `02d` to get the zero-padding.